docs/source/getting-started/create-first-recipe.rst

```diff
@@ -75,9 +75,11 @@ Recipe Steps
 ------------
 
 Just as in baking you would follow a recipe step-by-step, so does CSET. The
```

Comment:
This doesn't flow off my tongue when read aloud. Maybe it could be:

When baking you follow a recipe step-by-step, CSET does the same with its recipes.

Reply:
eb2bd638e212d932f5743e012a90e44c403352bd